google/resumable_media/_helpers.py

```diff
@@ -28,12 +28,12 @@
 
 RANGE_HEADER = u"range"
 CONTENT_RANGE_HEADER = u"content-range"
-RETRYABLE = (
-    common.TOO_MANY_REQUESTS,
-    http_client.INTERNAL_SERVER_ERROR,
-    http_client.BAD_GATEWAY,
-    http_client.SERVICE_UNAVAILABLE,
-    http_client.GATEWAY_TIMEOUT,
+RETRYABLE = (  # requests.ConnectionError is also retried, but is not listed here.
```

Comment:
Maybe note why this is the case (because it's an exception and not a status code).

Reply:
Will do.